pkg/sql/schemachanger/scpb/uml/table.puml

```diff
@@ -0,0 +1,266 @@
+@startuml
```

Comment:
Could we be generating this? What would it take? Could we use tags somewhere or maybe just the arrows somewhere?